owncloud/owncloud.py

```diff
@@ -1396,6 +1408,26 @@ def __check_ocs_status(tree, accepted_codes=[100]):
             r.status_code = int(code_el.text)
             raise OCSResponseError(r)
 
+    def make_ocs_request(self, method, service, action, **kwargs):
+        """Makes a OCS API request and analyses the response
+
+        :param method: HTTP method
+        :param service: service name
+        :param action: action path
+        :param \*\*kwargs: optional arguments that ``requests.Request.request`` accepts
+        :returns :class:`requests.Response` instance
+        """
+
+        accepted_codes = kwargs.get('accepted_codes', [100])
```

Comment:
Well that's tunneled through from this call to `self.__make_ocs_request(method, service, action, **kwargs)` which itself calls then request()